pkg/lock/lock_fast.go

```diff
@@ -24,6 +24,18 @@ type internalRWMutex struct {
 	sync.RWMutex
 }
 
+func (i *internalRWMutex) UnlockIgnoreTime() {
```

Comment:
Are these needed in the non-debug file? I don't see any references to them anywhere

Reply:
If anyone uses them in the code the compilation will fail to build for environments that don't are build with the `lockdebug` tag